tests/functional/api/groups/test_read.py

```diff
@@ -19,9 +19,9 @@ def test_it_returns_world_group(self, app):
         assert res.json[0]["id"] == "__world__"
 
     def test_it_returns_private_groups_along_with_world_groups(
-        self, app, factories, db_session, user_with_token, token_auth_header
+        self, app, factories, db_session, user, token_auth_header
     ):
-        user, _ = user_with_token
```

Comment:
The token from `user_with_token` was never used and can be replaced with `user`.